### PR TITLE
Add lost topics from the old documentation 

### PR DESCRIPTION
**Pull request by alexandraepan**
### Description 

Add two topics that were lost when reworking documentation from TBB to oneTBB. 

Fixes # - _issue number(s) if exists_

- [ ] - git commit message contains an appropriate signed-off-by string _(see [CONTRIBUTING.md](https://github.com/oneapi-src/oneTBB/blob/master/CONTRIBUTING.md#pull-requests) for details)_

### Type of change

_Choose one or multiple, leave empty if none of the other choices apply_

_Add a respective label(s) to PR if you have permissions_

- [ ] bug fix - _change that fixes an issue_
- [ ] new feature - _change that adds functionality_
- [ ] tests - _change in tests_
- [ ] infrastructure - _change in infrastructure and CI_
- [x] documentation - _documentation update_

### Tests

- [ ] added - _required for new features and some bug fixes_
- [x] not needed

### Documentation

- [ ] updated in # - _add PR number_
- [ ] needs to be updated
- [x] not needed

### Breaks backward compatibility
- [ ] Yes
- [x] No
- [ ] Unknown

### Notify the following users
@omalyshe @akukanov 

### Other information
Signed-off-by: Alexandra Epanchinzeva alexandra.epanchinzeva@intel.com

### CODE REVIEW

**Review by omalyshe — APPROVED** [review]
LGTM.

File: doc/main/tbb_userguide/Floating_Point_Settings.rst [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+If none of the outer-level contexts capture floating-point settings, the settings captured during task scheduler initialization are used.
+
+It guarantees that: 
+
+* Floating-point settings are applied to all tasks executed by the task scheduler, if they are captured: 
+
+  * To a task group context. 
+  * During task scheduler initialization. 
+
+* A call to the oneTBB parallel algorithm does not noticeably change the settings of the calling floating-point stream. Even if the algorithm is executed with different settings.
```
Comment (akukanov) [inline doc/main/tbb_userguide/Floating_Point_Settings.rst:47]: Please change to the following wording that is more technically correct

> An call to a oneTBB parallel algorithm does not change the floating-point settings of the calling thread, even if the algorithm uses different settings.

File: doc/main/reference/scalable_memory_pools/malloc_replacement_log.rst [hunk 2]
```diff
@@ -0,0 +1,84 @@
+.. _malloc_replacement_log:
+
+oneTBB_malloc_replacement_log Function
```
Comment (akukanov) [inline doc/main/reference/scalable_memory_pools/malloc_replacement_log.rst:3]: The function is called `TBB_malloc_replacement_log`.

File: doc/main/reference/scalable_memory_pools/malloc_replacement_log.rst [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+
+Description
+***********
+
+Dynamic replacement of memory allocation functions on Windows* OS uses in-memory binary instrumentation techniques. 
+To make sure that such instrumentation is safe, oneTBB first searches for a subset of replaced functions in the Visual C++* runtime DLLs
+and checks if each one has a known bytecode pattern. If any required function is not found or its bytecode pattern is unknown, the replacement is skipped, 
+and the program continues to use the standard memory allocation functions.
+
+The ``oneTBB_malloc_replacement_log`` function allows the program to check if the dynamic memory replacement happens and to get a log of the performed checks.
```
Comment (akukanov) [inline doc/main/reference/scalable_memory_pools/malloc_replacement_log.rst:37]: Same here, `TBB_malloc_replacement_log`

**Review by akukanov — COMMENTED** [review]
Sorry for a late after-the-commit review; but some changes are needed in the restored pages.

**Pull request merged**